modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryNetworkIssuesTest.java

```diff
@@ -262,9 +279,19 @@ private void doTestBackwardNodeCheckWithSameLoopback(String localhost) throws Ex
         Ignite node1 = startGrid(1);
 
         specialSpi = new TestDiscoverySpi();
+        testLog = testMethodLog;
+
+        int effAddrsSizeNode0 = spi(node0).getEffectiveNodeAddresses((TcpDiscoveryNode)node1.cluster().localNode()).size();
```

Comment:
Why we take addresses of node 1 if we want to kill node 0? Should we check backward?